config.toml

```diff
@@ -44,3 +44,6 @@ consumersecret    = ""
 accesstoken       = ""
 accesstokensecret = ""
 
+[youtube]
+key               = ""
```

Comment:
Where does this key come from? Is there a youtube specific key, or is it google account wide?